pbf/models.py

```diff
@@ -267,6 +268,10 @@ def elements(self):
         counter[Elements.Animal] += self.temporary_animal + self.permanent_animal
         for card in self.play.all():
             counter += card.get_elements()
+        if self.spirit.name == 'Earthquakes':
+            played_impending = GamePlayerImpendingWithEnergy.objects.filter(gameplayer=self, in_play=True)
+            for i in played_impending.all():
+                counter += i.card.get_elements()
```

Comment:
```suggestion
            played_impending = self.impending_with_energy.filter(gameplayerimpendingwithenergy__in_play=True)
            for i in played_impending.all():
                counter += i.get_elements()
```

You *could* potentially write it this way, but I understand that the resulting line is a little long (longer than the original), so I would understand if you didn't

Reply:
I'd prefer this way, just not familiar with Django 👍